### PR TITLE
fix(options.recredit): update recredit info message

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | DTRSD-57210
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~~Breaking change is mentioned in relevant commits~~ (not applicable)

### :bug: Bug Fises

cbb5c0d - fix(options.recredit): update recredit info message

### :flags: Translations

- [x] TRANS-39213

Signed-off-by: Antoine Leblanc <antoine.leblanc@ovhcloud.com>

### CODE REVIEW

**Review by cbourgois — DISMISSED** [review]

**Review by marie-j — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json [hunk 1]
```diff
@@ -2,8 +2,9 @@
   "sms_section_back_link": "Revenir aux options",
   "sms_options_recredit_title": "Recharge automatique",
   "sms_options_recredit_information_title": "Informations",
-  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.<br/>Cette recharge, qui s'effectue automatiquement, sera prélevée sur votre compte client OVH.",
-  "sms_options_recredit_information_note": "Veuillez noter que la recharge ne sera possible que si la balance de votre compte client OVH est positive ou nulle. Il vous est possible de créditer votre compte client dans votre Espace Client Télécom section Facturation.",
+  "sms_options_recredit_information_text": "Vous avez la possibilité de recharger automatiquement votre compte SMS lorsque votre solde franchit un certain seuil.",
+  "sms_options_recredit_information_note": "L’option de recharge automatique ne peut être activée qu'à partir de 2 mois d'ancienneté de votre service. Un moyen de paiement de type Prélèvement SEPA doit également être présent et valide sur votre compte OVHcloud.",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json:6]:
```suggestion
  "sms_options_recredit_information_note": "L’option de recharge automatique ne peut être activée qu'à partir de 2 mois d'ancienneté de votre service. Un moyen de paiement de type prélèvement SEPA doit également être présent et valide sur votre compte OVHcloud.",
```

File: packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -2,8 +2,9 @@
...
+  "sms_options_recredit_information_note2": "Cette recharge, qui s’effectue automatiquement, sera prélevée sur votre compte client OVHcloud.",
   "sms_options_recredit_manage_title": "Gérer les options",
   "sms_options_recredit_manage_threshold_label": "Seuil minimum :",
   "sms_options_recredit_manage_threshold_text": "{{ creditThresholdForAutomaticRecredit | number:2 }} crédits",
```
Comment (mohammed-zahaf) [inline packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json:10]: We can't use the angular filter (number:2) directly into template ?

File: packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json (hunk 1 above)
Comment (JeremyDec) [reply packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json:6]: @mohammed-zahaf, `Prélèvement SEPA` should be kept as a title form (capitalized). See for example: https://github.com/ovh/manager/commit/489b27bfd714edd65595b833780da751b9169899#diff-a52bb7ff0674c759d0861b68df01d40388d1938ad676fd81454b6bc3020dca67R2

File: packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json (hunk 2 above)
Comment (antleblanc) [reply packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json:10]: This part hasn't been touched (not the scope of this Pull Request).

AngularJS filters are used across multiple translation files according to following command output:
```sh
$ git grep '|' -- "**/Messages_fr_FR.json" | cut -d ':' -f 1 | uniq
packages/manager/apps/telecom/src/app/common/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/options/recredit/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/receivers/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/hlr/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/incoming/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/outgoing/translations/Messages_fr_FR.json
packages/manager/modules/sms/src/sms/sms/pending/translations/Messages_fr_FR.json
```

This is something that can be addressed via its own Pull Request I think.

Please let me know if you plan to address it.
Thanks!

**Review by mohammed-zahaf — DISMISSED** [review]

**Pull request merged**